spec/lib/embed/embed_this_panel_spec.rb

```diff
@@ -100,11 +99,6 @@
         expect(src).to match(/&hide_embed=true/)
         expect(src).to match(/&hide_title=true/)
       end
-
-      it 'does not include parameters that are not intended to be passed' do
```

Comment:
is this check no longer useful, or is "hide_metadata" no longer a good test string?